src/Tizen.Wearable.CircularUI.Forms/ContentButton.cs

```diff
@@ -0,0 +1,175 @@
+        /// </summary>
+        /// <since_tizen> 4 </since_tizen>
+        public View Content
+        {
+            get => (View)GetValue(ContentProperty);
+            set => SetValue(ContentProperty, value);
+        }
+
+        /// <summary>
+        /// Gets or sets command that is executed when the button is clicked.
+        /// </summary>
+        /// <since_tizen> 4 </since_tizen>
+        public ICommand Command
+        {
+            get => (ICommand)GetValue(CommandProperty);
+            set => SetValue(CommandProperty, value);
+        }
+
+        /// <summary>
+        /// Gets or sets command paramter that is executed when the button is clicked.
+        /// </summary>
+        /// <since_tizen> 4 </since_tizen>
+        public object CommandParameter
+        {
+            get => GetValue(CommandParameterProperty);
+            set => SetValue(CommandParameterProperty, value);
+        }
+
+        /// <summary>
+        /// Occurs when the button is clicked.
+        /// </summary>
+        /// <since_tizen> 4 </since_tizen>
+        public event EventHandler Clicked;
+
+        /// <summary>
+        /// Occurs when the button is pressed.
+        /// </summary>
+        /// <since_tizen> 4 </since_tizen>
+        public event EventHandler Pressed;
+
+        /// <summary>
+        /// Occurs when the button is released.
+        /// </summary>
+        /// <since_tizen> 4 </since_tizen>
+        public event EventHandler Released;
+
+        /// <summary>
+        /// For internal use.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool IsCommandEnabled
+        {
+            get => _isCommandEnabled;
+            set => _isCommandEnabled = value;
+        }
+
+        /// <summary>
+        /// For internal use.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void SendClicked()
+        {
+            if (IsCommandEnabled)
+                Command?.Execute(CommandParameter);
+
+            Clicked?.Invoke(this, EventArgs.Empty);
+        }
+
+        /// <summary>
+        /// For internal use.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void SendPressed()
+        {
+            Pressed?.Invoke(this, EventArgs.Empty);
+        }
+
+        /// <summary>
+        /// For internal use.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void SendReleased()
+        {
+            Released?.Invoke(this, EventArgs.Empty);
+        }
+
+        static void OnCommandChanged(BindableObject bindable, object oldCommand, object newCommand)
+        {
+            ContentButton button = (ContentButton)bindable;
+            if (newCommand is ICommand command)
+            {
+                button.IsCommandEnabled = command.CanExecute(button.CommandParameter);
+                command.CanExecuteChanged += button.OnCommandCanExecuteChanged;
```

Comment:
```suggestion
                command.CanExecuteChanged += button.OnCommandCanExecuteChanged;
                button.IsCommandEnabled = command.CanExecute(button.CommandParameter);
```
Need to call `CanExecute` after add `OnCommandCanExecuteChanged` handler

Reply:
Fixed